docs/tasks.md

```diff
@@ -502,6 +502,23 @@ spec:
       value: "http://google.com"
 ```
 
+Parameter declarations include default values which will be used if the parameter is not specified, for example to
```

Comment:
Is this sentence talking about default values for string and array parameters that are provided by Tekton or supplied by the user?

Reply:
supplied by the author of the Task/Pipeline - ill add a bit more text to try to make it more clear